src/client/__tests__/SignIn.test.tsx

```diff
@@ -41,7 +41,7 @@ test('Default terms and conditions in document when clientId not set', async ()
 		// Passing function to testing-library's `queryByText`
 		return queryByText((content, element) => {
 			const hasText = (element: Element | null) =>
-				element?.textContent === text;
+				element?.textContent?.startsWith(text) ?? false;
```

Comment:
Hey, do you mean to use startsWith here? ... I wonder if that is what you want to do whether it is worth renaming the function to something like `queryByTextContentPrefix`